nav2-usability/README.md

```diff
@@ -1 +1 @@
-This folder includes files for the [Navigator 2.0 API Usability Research](https://github.com/flutter/uxr/wiki/Navigator-2.0-API-Usability-Research). A report of this research has been made available in this folder. 
\ No newline at end of file
+This folder includes files for the [Flutter Routing API Usability Research](https://github.com/flutter/uxr/wiki/Flutter-Routing-API-Usability-Research). A report of this research has been made available in this folder. 
```

Comment:
We should name it to `Router API`, IMO Routing API is the super group of any API that is related to routing, which also includes Navigator API

Reply:
Yes, I did that when we refer to Router API specifically. The name of the project is more general, because we also evaluated all these packages instead of just Router.